File: Esame_22/JAVA/Laboratorio.java
import java.util.*;

public class Laboratorio extends Corso {
	// Attributi
	private String nomeLaboratorio;
	private String nomeAssistente;
	private int postazioni;

	// Costruttore
	public Laboratorio(int codiceCorso, String nomeCorso, String docente, String nomeLaboratorio, String nomeAssistente,
			int postazioni) {
		super(codiceCorso, nomeCorso, docente);
		this.nomeLaboratorio = nomeLaboratorio;
		this.nomeAssistente = nomeAssistente;
		this.postazioni = postazioni;
	}

	// Metodi
	// PUNTO 3: toString()
	public String toString() {
		return super.toString() + "lab" + "\t-\t-\t-\t" + this.nomeLaboratorio + "\t" + this.nomeAssistente + "\t"
				+ this.postazioni;
	}

} // fine classe

File: Esame_22/PYTHON/Laboratorio.py
from Corso import Corso

class Laboratorio(Corso): # classe Laboratorio che eredita da Corso

    # COSTRUTTORE
    def __init__(self, codiceCorso, nomeCorso, docente, nomeLaboratorio, nomeAssistente, postazioni ):
        super().__init__(codiceCorso, nomeCorso, docente)
        self._nomeLaboratorio = nomeLaboratorio
        self._nomeAssistente = nomeAssistente
        self._postazioni = postazioni

    # METODI
    def toString(self):
        return super().toString() + "lab" +  "\t-\t-\t-\t" + self._nomeLaboratorio + \
               "\t" + self._nomeAssistente + "\t" + str(self._postazioni)




